internal/storage/elasticsearch/config/config.go

```diff
@@ -486,6 +492,7 @@ func (c *Configuration) getConfigOptions(logger *zap.Logger) ([]elastic.ClientOp
 	if c.SendGetBodyAs != "" {
 		options = append(options, elastic.SetSendGetBodyAs(c.SendGetBodyAs))
 	}
+	options = append(options, elastic.SetGzip(c.HTTPCompression))
```

Comment:
SetGZip - does this mean ES supports other compression methods too? If yes then the flag name needs to be different, maybe compression methods too rather than just a Boolean.

Reply:
No, this is just the name of the corresponding setting in the olivere/elastic library, see [the documentation](https://github.com/olivere/elastic/wiki/Configuration)